src/App.js

```diff
@@ -204,9 +218,20 @@ class App extends React.Component {
             connectNotes={this.connectNotes}
           />
         )}
-      </>
+      </div>
     );
   }
+  
+  handleOnActive(event){
```

Comment:
arrow function notation already binds `this` by default, so ideally we can change this to arrow function and remove the binding in the constructor